### ISSUE TITLE
The ? menu is an accesibility issue. Change to &Help

### ISSUE THREAD

**Issue by koolraap** [notepad-plus-plus/notepad-plus-plus#5632]
The ? menu should be renamed to &Help so it's accessibility friendly. There's literally no upside to calling this menu ?

Help is the one thing you want people to be able to find without thinking.

(Please note I've raised this before and it was closed, IMO, incorrectly. I'm not raising this issue because I have accessibility issues. I'm raising it because I care about those that do. https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4601)

**Comment by dinkumoil** [comment]
I have to second that request. In the community forum people are frequently told to provide their debug infos but sometimes they are not able to find the right place to get it. I often thought if this might be caused due to the tiny question mark as a menu title or because they do not understand the description "Go to menu ? ...".

I think it would simplyfie giving support in the forum if the menu title would be `Help` as everyone expects.

**Comment by dinkumoil** [comment]
I have to second that request. In the community forum people are told to provide their debug info but sometimes they are not able to find the right place to get it. I often thought if this might be caused due to the tiny question mark as a menu title or because they do not understand the description "Go to menu ? ...".

I think it would simplyfie giving support in the forum if the menu title would be `Help` as everyone expects.

**Comment by dinkumoil** [comment]
I have to second that request. In the community forum people are told to provide their debug info but sometimes they are not able to find the right place to get it. I often thought if this might be caused due to the tiny question mark as a menu title or because they do not understand the description "Go to menu ? ...".

I think it would simplyfie giving support in the forum if the menu title would be `Help` as everyone expects.

**Comment by koolraap** [comment]
@sasumner no, I won't let it go. It's not a particularly divisive issue. "?" is a defect. You've argued before this is innovation because it's different. Innovation implies (IMO) better. ? is not better. Your role here appears to be a status quo apologist -- I look forward to you contributing an argument for the feature one way or another.

**Comment by PalashBansal** [comment]
IMHO, yes "?" sounds cool but aiming at the button is a headache, especially when I suck at first-person shooter games. and the shortcut button "alt+shift+/" too, as the standard keyboards doesn't have "?" button as primary but as "Shift+/"
On a side note: Why not "**Help?**" instead of "?" or "Help", with shortcut as "**Alt+H**"
We all want best out of npp, instead of heated up argument, lets @donho take up a call on this.

**Comment by sasumner** [comment]
How about the bigger point:  There isn't much actual help to be found on the `?` menu anyway.  All that's really there of substance is a way to copy "debug info" to the clipboard...

**Comment by DavidHansen-0x454d43** [comment]
@sasumner I disagree. While there is no link to an MHT helpfile, I count at least 4 different 'help' items under the '?' menu. (command line args, project page, forum, live support, etc).  

Comparing to visual studio's help menu, there is a similar amount on non-help items in this menu:
![image](https://user-images.githubusercontent.com/26116146/67334692-2f05a580-f4e8-11e9-85e5-6757d95d40ba.png)

To me, the "?" breaks my expectation of how a windows application should work. My expectation is that there should be a "help" menu entry, and it should be in the right most position of the menu-bar. 

To be innovate, the N++ should be better than other applications. However, I feel that the cumbersome keyboard accelerator and the lack of translation options means "?" is not "innovative".

**Comment by sasumner** [comment]
Yea, I meant help in terms of getting to the real documentation.  For sure the command-line args help is useful.  It's too bad that `?` doesn't appear in the language files as something that's translatable.
